online-classroom-management/src/main/resources/application.yml

```
@@ -19,3 +23,11 @@ spring:
     password:
     driver-class-name: com.mysql.cj.jdbc.Driver
 
+eureka:
+  instance:
+    prefer-ip-address: true
+  client:
+    fetch-registry: true
+    register-with-eureka: true
+    serviceUrl:
+      defaultZone: http://localhost:8070/eureka/
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)